### DISCUSSION TITLE
Embed logs viewer page to another page as a blade component

### DISCUSSION THREAD

**Discussion by zhainar** [opcodesio/log-viewer#404] [Ideas]
Is it possible to embed logs viewer page to another page as a blade component? I have orchid admin panel and I want to see logs there. It would be nice to have option to embed viewer to orchid screen because it already have its own gate and access control.

**Comment by arukompas** [comment]
Hey @zhainar , great idea, I thought about it as well, but it just isn't a priority at the moment. You're welcome to submit a PR if you have an idea of how to implement it!

The problem I see is the fact that Log Viewer is Vue.js based. So that Blade component would still have to somehow load the JS/CSS for Log Viewer.
